bin/console

```diff
@@ -2,13 +2,6 @@
 
 require "bundler/setup"
 require "applocale/ruby"
+require "pry"
 
-# You can add fixtures and/or initialization code here to make experimenting
-# with your gem easier. You can also use a different console, if you like.
-
-# (If you use this, don't forget to add pry to your Gemfile!)
-# require "pry"
-# Pry.start
-
-require "irb"
-IRB.start(__FILE__)
+Pry.start
```

Comment:
You don't like binding.irb? It's super handy haha.

Reply:
Never used binding.irb in my life but this is a preference and can easily be reverted.

Reply:
I'm okay leaving pry in there as its pretty common.